k8sutils/statefulset_test.go

```diff
@@ -284,6 +284,86 @@ func Test_createStatefulSet(t *testing.T) {
 	}
 }
 
+func TestUpdateStatefulSet(t *testing.T) {
+	tests := []struct {
+		name        string
+		sts         appsv1.StatefulSet
```

Comment:
Create 2 statefulset one is existing and other is updated.
Your existing one should be passed to the `k8sClientFake.NewSimpleClientset()`.